system/qcomgpsd/qcomgpsd.py

```diff
@@ -101,29 +102,23 @@ def gps_enabled() -> bool:
     raise Exception("failed to execute QGPS mmcli command") from exc
 
 def download_assistance():
+  http = urllib3.PoolManager()
```

Comment:
any reason you switched to urllib? the requests version of this looked a bit nicer

Reply:
just PR consistency, but I agree, reverted